### DISCUSSION TITLE
Welcome to OpenAPI.Net Discussions!

### DISCUSSION THREAD

**Discussion by amusleh-spotware-com** [spotware/OpenAPI.Net#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Husnainmirza0071** [comment]
Thanks for everything

**Comment by Husnainmirza0071** [comment]
Thanks for everything

**Comment by axax679** [comment]
Install-Package Spotware.OpenAPI.Net

**Comment by axax679** [comment]
j

**Comment by dualwieldingrogue** [comment]
Isn't this Deprecated?
